### PR TITLE
Update deployment target definition and best practices

### PR DESCRIPTION
**Pull request by BobJWalker**

### CODE REVIEW

File: docs/getting-started/best-practices/installation-guidelines.md [hunk 1]
```diff
@@ -26,28 +26,50 @@ This document will refer to tasks; a task can be:
...
 
+Of that list, Deployments and Runbook Runs are the most common.
+
 Octopus Deploy installation requirements are:
 - SQL Server 2016 or higher (AWS RDS SQL Server and Azure SQL are supported)
 - Windows Server 2012 R2 or later when hosting on Windows Server
 - Linux is supported when using the [Octopus Deploy Linux Docker image](https://octopus.com/blog/introducing-linux-docker-image)
 
-[High availability](/docs/administration/high-availability/index.md) functionality is included with both Server and Data Center licenses.  
+[High availability](/docs/administration/high-availability/index.md) functionality may or may not be included in your license.  All Server, 2022+ or new Professional and Enterprise licenses include at least 2 nodes.
```
Comment (ryanrousseau) [inline docs/getting-started/best-practices/installation-guidelines.md:36]: What is 2022+ in this context?

File: docs/getting-started/best-practices/installation-guidelines.md [hunk 2]
```diff
@@ -26,28 +26,50 @@ This document will refer to tasks; a task can be:
...
+
+We recommend using a Production deployment window to determine how many tasks you'll need.  The eventual goal of CI/CD is to provide the capability to deploy to Production in the middle of the day.  However, in practice, depending on the application and users, that may not feasible nor desirable.  The Production deployment window is when all deployments _must complete_.  
+
+You should consider:
+
+- The duration of the window
+- How many applications need to be deployed during the window
+- How long, on average, each deployment will take.  If you are unsure, we recommend starting with 30 minutes.
+
+Knowing those three items, you can calculate the number of "deployment minutes" required.  Deploying 50 applications, each taking 30 minutes during the deployment window, will require 1,500 "deployment minutes."  Then we can divide that by the duration of the window, for example, two hours or 120 minutes.  1,500 divided by 120 is 12.5, which is the minimum task cap.  That is making a number of assumptions, everything goes perfectly, the deployments have no dependencies, if there are dependencies, it is low.  A reasonable task cap would be 15 to 20.
```
Comment (ryanrousseau) [inline docs/getting-started/best-practices/installation-guidelines.md:70]: > That is making a number of assumptions, everything goes perfectly, the deployments have no dependencies, if there are dependencies, it is low. 

This sentence is hard to follow.

File: docs/getting-started/best-practices/installation-guidelines.md [hunk 3]
```diff
@@ -86,16 +108,16 @@ For the remainder of this document, the assumption is you will be using Windows
 
 A high availability configuration will involve setting up:
 
-- 1 to 3 Windows servers, each with 2 cores / 4 GB of RAM with the task cap set to 10 for each server.
+- 1 to 3 Windows servers, each with 2 cores / 4 GB of RAM with the task cap set to 10-15 for each server.
```
Comment (ryanrousseau) [inline docs/getting-started/best-practices/installation-guidelines.md:111]: Should we recommend a single value (15) here instead of a range?

File: docs/infrastructure/deployment-targets/kubernetes-target/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ position: 50
 ---
 Kubernetes targets are used by the [Kubernetes steps](/docs/deployments/kubernetes/index.md) to define the context in which deployments and scripts are run.
 
-Conceptually, a Kubernetes target represent a permission boundary and an endpoint. Kubernetes [permissions](https://oc.to/KubernetesRBAC) and [quotas](https://oc.to/KubernetesQuotas) are defined against a namespace, and both the account and namespace are captured as a Kubernetes target, along with the cluster endpoint URL.
+Conceptually, a Kubernetes target represent a permission boundary and an endpoint. Kubernetes [permissions](https://oc.to/KubernetesRBAC) and [quotas](https://oc.to/KubernetesQuotas) are defined against a namespace, and both the account and namespace are captured as a Kubernetes target, along with the cluster endpoint URL.  The deployment target defines the default namespace, the namespace can be overwritten in the deployment process.
```
Comment (ryanrousseau) [inline docs/infrastructure/deployment-targets/kubernetes-target/index.md:8]: I think this can use some of the other wording you had landed on:

> A namespace is required when registering the Kubernetes cluster with Octopus Deploy.  By default, the namespace used in the registration is used in health checks and deployments.  The namespace can be overwritten in the deployment process.

File: docs/getting-started/best-practices/installation-guidelines.md (hunk 1 above)
Comment (BobJWalker) [reply docs/getting-started/best-practices/installation-guidelines.md]: Clarified.

File: docs/getting-started/best-practices/installation-guidelines.md (hunk 2 above)
Comment (BobJWalker) [reply docs/getting-started/best-practices/installation-guidelines.md]: Good shout, I've re-written it.

File: docs/getting-started/best-practices/installation-guidelines.md (hunk 3 above)
Comment (BobJWalker) [reply docs/getting-started/best-practices/installation-guidelines.md]: Yup, I've set it to 15 and adjusted the bits below that.

File: docs/infrastructure/deployment-targets/kubernetes-target/index.md (hunk 4 above)
Comment (BobJWalker) [reply docs/infrastructure/deployment-targets/kubernetes-target/index.md]: Good shout, I've updated it.

**Review by ryanrousseau — APPROVED** [review]
